Toolkit concept docs XAML: batch 1 (#554)
Esri/arcgis-maps-sdk-dotnet-toolkit: docs/basemap-gallery.md

## Usage

Ensure that your `GeoModel` is not null before selecting a basemap with the `BasemapGallery`.

### .NET MAUI:

```xml

> <Grid ColumnDefinitions="*,300">

Reviewer: I like the more succinct XAML 👍

    <esri:MapView x:Name="mapView" />
    <toolkit:BasemapGallery Grid.Column="1" GeoModel="{Binding Source={x:Reference mapView}, Path=Map}"/>
</Grid>
```

### UWP/WinUI:

```xml
<Grid>
    <Grid.ColumnDefinitions>
        <ColumnDefinition Width="*" />
        <ColumnDefinition Width="300" />
    </Grid.ColumnDefinitions>
    <esri:MapView x:Name="mapView" />
    <toolkit:BasemapGallery Grid.Column="1" GeoModel="{Binding ElementName=mapView, Path=Map}" />
</Grid>
```

### WPF:

> The usage in WPF is identical to UWP/WinUI minus one important distinction. The `BasemapGallery` should be accessed with the same prefix as the `GeoView`. 

Reviewer: Minor tweaks to make it more direct ("should be" => "is", eg).
```suggestion
The usage in WPF is identical to UWP/WinUI with one important distinction: `BasemapGallery` is accessed with the same namespace prefix as `GeoView`. 
```
